### ISSUE TITLE
is circuit breaker threadsafe?

### ISSUE THREAD

**Issue by madyar997** [sony/gobreaker#55]
Do I need to use a mutex when calling cb.Execute from multiple goroutines?
as it seems, counts won't be calculated correctly and need to be synchronized

**Comment by GingerMoon** [comment]
“ gobreaker is goroutine-safe. You can run gobreaker in parallel.“
reference:
https://github.com/sony/gobreaker/issues/27

**Comment by vladimirshishmintsev-ht** [comment]
@madyar997 you are right. That's not about counters exactly. That's how and when breakers registers an error and manages the queue of parallel requests. In two words - they delegates it to you.
I registered the new issue: https://github.com/sony/gobreaker/issues/91
